packages/quantic/force-app/main/default/lwc/quanticFacetValue/quanticFacetValue.js

```diff
@@ -58,6 +60,13 @@ export default class QuanticFacetValue extends LightningElement {
     return new Intl.NumberFormat(LOCALE).format(this.item.numberOfResults);
   }
 
+  get divRole (){
+    return this.displayAsLink ? "button":"checkbox"
+  }
+  get ariaLabelVoiceOver(){
+    return `Filter, ${this.formattedFacetValue}, ${this.numberOfResults}`
```

Comment:
@acarpentier01 This is the text that will be read by a screen reader if that makes sense:
@adroletCoveo This is an example of what I was talking about.

Example: 🔊 "Filter, pdf, 1234"

Reply:
> @acarpentier01 This is the text that will be read by a screen reader if that makes sense: @adroletCoveo This is an example of what I was talking about.
> 
> Example: 🔊 "Filter, pdf, 1234"
not a issue, set to resolved.